java/ql/src/experimental/Security/CWE/CWE-094/GroovyInjection.ql

```diff
@@ -0,0 +1,21 @@
+/**
+ * @name Groovy Language injection
+ * @description Evaluation of a user-controlled Groovy script
+ *              may lead to arbitrary code execution.
+ * @kind path-problem
+ * @problem.severity error
+ * @precision high
+ * @id java/groovy-injection
+ * @tags security
+ *       external/cwe/cwe-094
+ */
+
+import java
+import semmle.code.java.dataflow.FlowSources
```

Comment:
I believe there is no need to import FlowSources as `GroovyInjectionLib` imports it.